powerlevel9k.zsh-theme

```diff
@@ -1317,8 +1317,8 @@ prompt_powerlevel9k_setup() {
 
   # Display a warning if the terminal does not support 256 colors
   local term_colors
-  term_colors=$(echotc Co)
-  if (( $term_colors < 256 )); then
+  term_colors=$(echotc Co 2>/dev/null)
+  if (( ! $? && ${term_colors:-0} < 256 )); then
```

Comment:
Are you suggesting that the `if` condition be simplified to

```zsh
if (( ${term_colors:-0} < 256 )); then
```

That will still print the warning if `echotc Co` failed. However, we could use this instead:

```zsh
if (( ${term_colors:-256} < 256 )); then
```

That's more concise, though perhaps also more obscure. I can certainly revise my pull request to use that approach if you prefer it. Just let me know.